monkey/monkey_island/cc/test_encryptor.py

```diff
@@ -0,0 +1,37 @@
+import os
+
+from monkey_island.cc.consts import MONKEY_ISLAND_ABS_PATH
+from monkey_island.cc.encryptor import initialize_encryptor, encryptor
+
+
+TEST_DATA_DIR = os.path.join(MONKEY_ISLAND_ABS_PATH, "cc", "testing")
+PASSWORD_FILENAME = "mongo_key.bin"
+
+PLAINTEXT = "Hello, Monkey!"
+CYPHERTEXT = "vKgvD6SjRyIh1dh2AM/rnTa0NI/vjfwnbZLbMocWtE4e42WJmSUz2ordtbQrH1Fq"
+
+
+def test_aes_cbc_encryption():
+    initialize_encryptor(TEST_DATA_DIR)
+
+    assert encryptor().enc(PLAINTEXT) != PLAINTEXT
+
+
+def test_aes_cbc_decryption():
+    initialize_encryptor(TEST_DATA_DIR)
+
+    assert encryptor().dec(CYPHERTEXT) == PLAINTEXT
```

Comment:
I think more important than both of these cases would be to test `assert encryptor().dec(encryptor().enc(PLAINTEXT)) == PLAINTEXT`. Currently, if encryption fails, but still generates something your test would pass.

Reply:
I waffled back and forth on this. I'll add it.